app/controllers/public_recipes_controller.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class PublicRecipesController < ApplicationController
+  def index
```

Comment:
- [ ] Kindly display all public recipes for all users not related to the current user and should ordered by the newest as in the wireframe as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md#:~:text=ordered%20by%20newest%20as%20in%20the%20wireframe.).
- [ ] Kindly make sure that recipes in the public recipe lead to the `recipe details` as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md#:~:text=newest%20as%20in%20the%20wireframe.-,Should%20lead%20to%20recipe%20details,-.).

Reply:
- [ ] Just as the previous reviewer mentioned Kindly make sure that recipes in the public recipe lead to the recipe details as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md#:~:text=newest%20as%20in%20the%20wireframe.-,Should%20lead%20to%20recipe%20details,-.).